cc_launcher.sh

```
@@ -0,0 +1,888 @@
+        
+        retry_count=$((retry_count + 1))
+    done
+    
+    print_error "无法获取线上版本信息（已重试 $max_retries 次）"
+    return 1
+}
+
+# 下载并更新脚本
+update_script() {
+    local script_path="$0"
+    local temp_file=$(mktemp)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)